addons/point_of_sale/static/src/app/screens/receipt_screen/receipt_screen.js

```diff
@@ -53,23 +53,13 @@ export class ReceiptScreen extends Component {
     get nextScreen() {
         return { name: "ProductScreen" };
     }
-    get ticketScreen() {
-        return { name: "TicketScreen" };
-    }
     orderDone() {
         this.currentOrder.uiState.screen_data.value = "";
         this.currentOrder.uiState.locked = true;
         this._addNewOrder();
         const { name, props } = this.nextScreen;
         this.pos.showScreen(name, props);
     }
-    resumeOrder() {
-        this.currentOrder.uiState.screen_data.value = "";
-        this.currentOrder.uiState.locked = true;
-        this.pos.selectNextOrder();
-        const { name, props } = this.ticketScreen;
-        this.pos.showScreen(name, props);
-    }
```

Comment:
this option is not needed anymore. If a customer wants to go to another draft order, they can just click on that orders tab in the topbar.